docs/release-notes/release-notes-0.15.0.md

```diff
@@ -72,6 +72,9 @@ releases. Backward compatibility is not guaranteed!
   caveats/restrictions to be added to an existing macaroon (instead of needing
   to bake a new one). 
 
+* [Add `payment_addr` flag to `buildroute`](https://github.com/lightningnetwork/lnd/pull/6576)
```

Comment:
Fat fingered...realized this should target 0.15.1, will push up a small commit to move it over.